content/sensu-go/5.15/api/overview.md

```diff
@@ -135,6 +135,93 @@ http://127.0.0.1:8080/api/core/v2/namespaces/default/events
 If your token expires, you should see a 401 Unauthorized response from the API.
 To regenerate a valid access token, first run any sensuctl command (like `sensuctl event list`) then repeat step 2.
 
+### Authenticate with the API key feature
+
+The Sensu API key feature (core/v2.APIKey) is a persistent UUID that maps to a stored Sensu username. API keys are cluster-wide resources, so only cluster admins can grant, view, and revoke them.
```

Comment:
```suggestion
The Sensu API key feature (core/v2.APIKey) is a persistent UUID that maps to a stored Sensu username. The advantages of authenticating with API keys rather than [access tokens](#authentication-quick-start) include:
- **More efficient integration**: Check and handler plugins and other code can integrate with the Sensu API without implementing the logic required to authenticate via the `/auth` API endpoint to periodically refresh the access token.
- **Improved security**: API keys do not require providing a username and password in check or handler definitions.
- **Better admin control**: API keys can be created and revoked without changing the underlying user's password...but keep in mind that API keys will continue to work even if the user's password changes.

API keys are cluster-wide resources, so only cluster admins can grant, view, and revoke them.
```

Reply:
What do you think of ^?

Reply:
Let's copy these comments to the guide and :shipit:

Reply:
Done! Thank you.